smart_contracts/contract/src/contract_api/storage.rs

```diff
@@ -336,6 +336,30 @@ pub fn disable_contract_version(
     api_error::result_from(result)
 }
 
+/// Enable a version of a contract from the contract stored at the given
+/// `Key`. That version of the contract will no longer be callable by
+/// `call_versioned_contract`. Note that this contract must have been created by
+/// `create_contract` or `create_contract_package_at_hash` first.
```

Comment:
`create_contract` is a private function.  I guess this should be:

```suggestion
/// [`new_contract`] or [`create_contract_package_at_hash`] first.
```

I think this change applies to a few other doc comments in this file.